### ISSUE TITLE
Truncated Distributions

### ISSUE THREAD

**Issue by assignUser** [kgoldfeld/simstudy#78]

**Comment by assignUser** [comment]
I think  it would be nicer to add another field/parameter for the truncated distributions rather than doubling up all distributions as `dist_trunc`. It would maybe be a good time to do the switch to parameter names without inherent meaning, but this would be potentially a breaking change (we could possibly implement it in a way that is backwards compatible but I am not sure how well that would work). So we should think about this carefully.

**Comment by kgoldfeld** [comment]
I agree - it is nice to have a motivating reason to do this, other than the fact that using "variance" to refer to a dispersion parameter or, even worse, the number of trials in a binomial trial seems reason enough.

As far as implementing the change to param1/param2/param3, seems like a perfect kind of challenge for you - how to transition gracefully from ill-considered non-professional interface to a more professional one.

**Comment by assignUser** [comment]
And another issue to consider when designing the changes for #75 (I'm going through and linking these issues so we don't lose track :) )
